inventory/templates/inventory/item_detail.html

```diff
@@ -50,6 +50,10 @@ <h6>
             {% endautoescape %}
             {% endif %}
 
+            {% if not item.can_loan%}
+            <p> Hvis du vil låne denne gjenstanden kontakt ledelsen her: <b> hackerspace-styret@idi.ntnu.no </b></p>
```

Comment:
Could you change `ledelsen` to `styret` as the mail suggests and also make the email a link with `<a>` tags. To link to a mail you can do `href="mailto:hackerspace-styret@idi.ntnu.no"`